README.md

```diff
@@ -32,41 +32,33 @@ From the project root directory, download the Hyperledger Fabric peer binaries b
 Note that downloading the binaries will take some time.
 
 In the `webapp/server/.env` file, replace the `ORDERER_NAME`, `PEER_NAME`, and `MSP_ID` variables with:
-
-* `ORDERER_NAME` — the orderer name of the network you deployed your node in. To access the orderer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric network, click **Details**, copy the **Orderer name** value.
-* `PEER_NAME` — the peer name of the node that you deployed. To access the peer name—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **Peer name** value.
-* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the node that you deployed. To access the MSP ID—in the [Chainstack UI](https://console.chainstack.com/), navigate to your Hyperledger Fabric node, copy the **MSP ID** value.
+* `ORDERER_NAME` — the orderer name of the network you deployed your peer in. To access the orderer name, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, select **Service nodes** tab and click on **Orderer** to access its details page. Here you can copy the **Orderer name** value.
+* `PEER_NAME` — the name of the peer that you deployed. To access the peer name, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **Peer name** value.
+* `MSP_ID` — the Membership Service Provider identity (MSP ID) of the peer that you deployed. To access the MSP ID, in the [Chainstack UI](https://console.chainstack.com/), from Hyperledger Fabric network, select **Peer nodes** tab, click on your peer name to access its details page. Here you can copy the **MSP ID** value.
 
 You are now ready for the bootcamp.
 
 ### Set up your environment (optional)
 
 Additionally, you may want to prepare the web app client and server in advance—although this is not required, as this will be done during the bootcamp.
 
-In the `webapp` directory, create a `certs` directory.
-
-In the [Chainstack UI](https://console.chainstack.com/):
-
-* Navigate to your Hyperledger Fabric network, click **Details**. Click **Export orderer TLS certificate**. Click **Export connection profile**.
-* Navigate to your Hyperledger Fabric node. Next to **Organization identity**, click **Export**. Unzip the downloaded file.
-
-Copy the three exported files to the `webapp/certs/` directory you created previously.
+1. Export the [required files](./webapp/certs/README.md) from [Chainstack UI](https://console.chainstack.com/).
```

Comment:
Why move the export instructions to a separate file?

Reply:
I created a readme file in the certs folder so the participants don't have to create a certs folder.
Hence i thought it made more sense to move the instructions there.